proxy/enfuse/rpc/fuse.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package enfuse;
```

Comment:
How are we choosing proto package names? All other things being equal, I think I'd prefer one more closely related to the filepath, so `proxy.enfuse.rpc`, or `proxy.enfuse` if an `rpc` directory is convention.

Reply:
ah go the package name of the *.proto files actually gets passed in as the go option when it's autogenerated. So I just changes it to fusepb so that autocompleted picks it up as the package name.